### PR TITLE
[MIG] account_payment_batch_process

### PR DESCRIPTION
**Pull request by faryrosa**
#161 
[MOD]Including translation for Dominican Republic
[MOD]The amount to text calculation is replaced
[MOD]Change in the search for keys in the dictionary
[MOD]Setting in the default_get method call in AccountRegisterPayments
[MOD]Adjust in the view to guarantee read-only fields storage

### CODE REVIEW

**Comment by faryrosa** [comment]
Okay, I'm working on the changes. Thank you

**Comment by pedrobaeza** [comment]
Can you put a more appropriated title and what this does?

**Comment by faryrosa** [comment]
When I make the changes I modify the description with your suggestions

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/home/afuentes/py3_new/bin/python"
+}
```
Comment (hbto) [inline .vscode/settings.json]: This folder/file includes absolute paths to your folders.

Regards

File: account_payment_batch_process/README.rst [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit https://odoo-community.org.
```
Comment (max3903) [inline account_payment_batch_process/README.rst:36]: @faryrosa Can you use the new readme structure from:
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

**Review by max3903 — CHANGES_REQUESTED** [review]

**Review by bodedra — CHANGES_REQUESTED** [review]
Please respect commit history: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0

**Comment by kittiu** [comment]
The name should be like [11.0][MIG] account_payment_batch_process ?

**Comment by kittiu** [comment]
@faryrosa would you also add more test? :)

**Comment by faryrosa** [comment]
Hello

I apologize, but I could not advance more in this migration.

**Comment by max3903** [comment]
Thanks @faryrosa. @murtuzasaleh will take it from here.

**Comment by murtuzasaleh** [comment]
> Thanks @faryrosa. @murtuzasaleh will take it from here.

#255

**Pull request closed without merging**